py/bitbox02/devices.py

```diff
@@ -24,6 +24,7 @@
 BB02_BOOTLOADER = "bb02-bootloader"
 BB02BTC_BOOTLOADER = "bb02btc-bootloader"
 BITBOX02 = "BitBox02"
+BITBOX02BTC = "BitBox02BTC"
```

Comment:
I think we should avoid exposing these variables as our external API and instead have one function per device type.